hazelcast/src/main/java/com/hazelcast/jet/Job.java

```diff
@@ -172,13 +173,13 @@ default String getName() {
     JobConfig getConfig();
 
     /**
-     * Sets the specified configuration if this job is suspended.
+     * Applies the specified delta configuration if this job is suspended.
      *
      * @throws IllegalStateException if this job is not suspended
      * @throws UnsupportedOperationException if called for a light job
      * @since 5.3
      */
-    void setConfig(@Nonnull JobConfig config);
+    void updateConfig(@Nonnull DeltaJobConfig deltaConfig);
```

Comment:
operation in client-protocol returns `Data` (`updateJobConfig`) but here it is void